lineage/tHMM.py

```diff
@@ -19,19 +20,20 @@ def __init__(self, X: list[LineageTree], nState: int, fpi=None, fT=None, fE=None
         :param X: A list of objects (cells) in one lineage
         :param nStates: The number of hidden states
         """
+
         self.fpi = fpi
         self.fT = fT
         self.fE = fE
         self.num_states = nState
 
         if self.fpi is None or self.fpi is True:
-            self.pi = np.random.rand(nState)
+            self.pi = sp.uniform.rvs(0.0, 1.0, nState)
```

Comment:
This isn't using the random number generator. You can use [`rng.uniform()`](https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.uniform.html) here.